charts/renku/values.yaml

```diff
@@ -203,155 +203,47 @@ keycloak:
-  # postgresConfig:
-
-  ## Persist data to a persistent volume
-  persistence:
-    enabled: true
 
-    ## A manually managed Persistent Volume and Claim
-    ## Requires persistence.enabled: true
-    ## If defined, PVC must be created manually before volume will be bound
-    # existingClaim:
+  ## We use the defaults here. Note that these basic configs could also be set as
+  ## global values such that sub-charts can access them too.
+  postgresqlDatabase: postgres
+  postgresqlUsername: postgres
 
-    ## database data Persistent Volume Storage Class
-    ## If defined, storageClassName: <storageClass>
-    ## If set to "-", storageClassName: "", which disables dynamic provisioning
-    ## If undefined (the default) or set to null, no storageClassName spec is
-    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
-    ##   GKE, AWS & OpenStack)
-    ##
-    # storageClass: "-"
-    accessMode: ReadWriteOnce
-    size: 8Gi
-    subPath: postgresql-db
-    mountPath: /var/lib/postgresql/data/pgdata
+  ## The admin password should be set explicitly, otherwise a random string will be
+  ## created. Alternatively an existing secret can be provided. Note that postgres
+  ## DOES NOT tolerate a change of the admin password when upgrading.
+  # postgresqlPassword:
 
-    # annotations: {}
+  ## Use an existing secret instead of creating a new one. It must have a
+  ## postgresql-password key containing the password for the posgres user.
+  # existingSecret:
 
-  metrics:
-    enabled: false
-    image: wrouesnel/postgres_exporter
-    imageTag: v0.1.1
-    imagePullPolicy: IfNotPresent
-    resources:
-      requests:
-        memory: 256Mi
-        cpu: 100m
-      ## Define additional custom metrics
-      ## ref: https://github.com/wrouesnel/postgres_exporter#adding-new-metrics-via-a-config-file
-      # customMetrics:
-      #   pg_database:
-      #     query: "SELECT d.datname AS name, CASE WHEN pg_catalog.has_database_privilege(d.datname, 'CONNECT') THEN pg_catalog.pg_database_size(d.datname) ELSE 0 END AS size FROM pg_catalog.pg_database d where datname not in ('template0', 'template1', 'postgres')"
-      #     metrics:
-      #       - name:
-      #           usage: "LABEL"
-      #           description: "Name of the database"
-      #       - size_bytes:
-      #           usage: "GAUGE"
-      #           description: "Size of the database in bytes"
-
-  ## Configure resource requests and limits
-  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
-  ##
-  resources:
-    requests:
-      memory: 256Mi
-      cpu: 100m
-
-  service:
-    type: ClusterIP
-    port: 5432
-    externalIPs: []
-    ## Manually set NodePort value
-    ## Requires service.type: NodePort
-    # nodePort:
-
-  networkPolicy:
-    ## Enable creation of NetworkPolicy resources.
-    ##
-    enabled: false
+  image:
+    repository: bitnami/postgresql
+    tag: 9.6
```

Comment:
not newer?

Reply:
I was considering, but 9.6 is supported until November 2021, and migrating to 10 (or even 11) would yet be another stunt. At the same time I think it's important to get the chart update out ASAP as our current chart with the current postgres dependency does not run on recent k8s versions. So I'd prefer to defer this upgrade.